config.js

```diff
@@ -84,7 +84,7 @@ export default {
             },
             "www.google-analytics.com": {
                 urlMatch: /\/collect/,
-                queryParameterName: "uip"
+                queryParameterName: "_uip"
```

Comment:
Please don't remove `uip`; by removing it you will immediately drop UA support. Just like in the example above, it should be this:

```
queryParameterName: ["uip", "_uip"]
```

I.e. it will append both `uip` and `_uip` to all `/collect` requests to `www.google-analytics.com`.

Reply:
Make sense, fixed